### ISSUE TITLE
Problem with running the CxSM example

### ISSUE THREAD

**Issue by mb-1054571817** [DR-algo/DRalgo#11]
Hi!

I seem to have trouble with running the CxSM example in the file "cxsm.m". After running the cell
"ImportModelDRalgo[Group,gvvv,gvff,gvss,\[Lambda]1,\[Lambda]3,\[Lambda]4,\[Mu]ij,\[Mu]IJ,\[Mu]IJC,Ysff,YsffC,Verbose->False];
PerformDRhard[]"
I get the following error message:
"SparseArray::list:List expected at position 1 in [...]".

Subsequent cells also give an error message, but I don't know if those are independent issues or if they are related to the first one.

Is there a bug in the code, or could there be an issue with my setup? The example files "ah.m" and "ah-thermo.m" seem to work fine.

Thanks in advance!

**Comment by AndreasEkstedt** [comment]
Hello mb-1054571817, thanks for your message. Since the example runs fine on my system, can you please provide the mathematica and DRalgo versions that you are using?
Also, make sure that the entire "Model" tab is evaluated before running "ImportModelDRalgo[..."

Kind regards,
Andreas

**Comment by mb-1054571817** [comment]
Thanks for your feedback! 

I had the same trouble earlier today when running the file, but after restarting Mathematica once more it now seems to work. I do not know why it did not work before. I had tried several times (after restarting Mathematica) before opening this issue.

I am running Mathematica 13.0. I do not know the version of DRalgo, but I did a fresh installation from github a few days ago, so I believe I have the latest version.

I will close the issue since it appears that the problem was at my end.
